packages/component-library/stories/Select.story.js

```diff
@@ -0,0 +1,160 @@
+                  }}
+                  value={get("value")}
+                  disabled={disabled}
+                  options={options}
+                />
+              );
+            }}
+          </StatefulWrapper>
+        );
+      },
+      { notes }
+    )
+    .add(
+      "Standard Two",
+      () => {
+        const formLabel = text(
+          "Form label",
+          "TriMet Bus Lines",
+          GROUP_IDS.LABELS
+        );
+        const disabled = boolean("Disabled", false, GROUP_IDS.STATE);
+        const optionsOne = [
+          { value: 1, label: "All" },
+          { value: 2, label: "2 - Division" },
+          { value: 4, label: "4 - Fessenden" },
+          { value: 5, label: "6 - Martin Luther King Jr. Blvd" },
+          { value: 8, label: "8 - Jackson Park/NE 15th" },
+          { value: 9, label: "9 - Powell Blvd" },
+          { value: 10, label: "10 - Harold St" }
+        ];
+        const options = object("List items", optionsOne, GROUP_IDS.DATA);
+        return (
+          <StatefulWrapper initialState={{ value: 1 }}>
+            {({ get, set }) => {
+              return (
+                <Select
+                  formLabel={formLabel}
+                  onChange={event => {
+                    set({ value: event.target.value });
+                    action("onChange")(event);
+                  }}
+                  value={get("value")}
+                  disabled={disabled}
+                  options={options}
+                />
+              );
+            }}
+          </StatefulWrapper>
+        );
+      },
+      { notes }
+    )
+    .add(
+      "Example: Form group",
+      () => {
+        const formLabel = text(
+          "Form label",
+          "TriMet Bus Lines",
+          GROUP_IDS.LABELS
+        );
+        const formHelperText = text(
+          "Form helper text",
+          "* Select a bus line",
+          GROUP_IDS.LABELS
+        );
+        const disabled = boolean("Disabled", false, GROUP_IDS.STATE);
+        const optionsOne = [
+          "All",
+          "2 - Division",
+          "4 - Fessenden",
+          "6 - Martin Luther King Jr. Blvd",
+          "8 - Jackson Park/NE 15th",
+          "9 - Powell Blvd",
+          "10 - Harold St"
+        ];
+        const options = object("List items", optionsOne, GROUP_IDS.DATA);
+        return (
+          <StatefulWrapper initialState={{ value: "All" }}>
+            {({ get, set }) => {
+              return (
+                <Fragment>
+                  <Select
+                    formLabel={formLabel}
+                    onChange={event => {
+                      set({ value: event.target.value });
+                      action("onChange")(event);
+                    }}
+                    value={get("value")}
+                    disabled={disabled}
+                    options={options}
+                  />
+                  <FormHelperText>{formHelperText}</FormHelperText>
```

Comment:
FormHelperText should be implemented in Select.js by adding an optional `helperText` prop -- One of our goals is that the developer using our component shouldn't have to know anything about Material to use our component.